runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -24,10 +24,15 @@
  */
 package net.runelite.client.plugins.idlenotifier;
 
+import java.awt.Color;
+import net.runelite.client.Notifier;
+import net.runelite.client.config.Alpha;
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
+import java.awt.*;
```

Comment:
Don't use star imports for non-static imports. Please [review our Code Contribution guidelines](https://github.com/runelite/runelite/wiki/Code-Conventions).